eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -560,12 +560,17 @@ function Search-GitHubIssues {
     [ValidateNotNullOrEmpty()]
     [Parameter(Mandatory = $true)]
     $CommitHash,
-    $State="open"
+    $State="open",
+    [ValidateNotNullOrEmpty()]
+    [Parameter(Mandatory = $true)]
```

Comment:
Do we need this to be mandatory? I would suggest if it isn't set we don't pass the header.